packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/about.a.tool.mdx

```diff
@@ -1,123 +1,73 @@
-
-```bash
-npm run start
-```
+## Adding content
 
-You will see your readme hosted on the site. Next lets add some pages.
+The TerraDevSite Plugin will search the `terra-dev-site` folder for files with a defined content extension, typically `test` or `doc`. The default export from those files will be imported into the generated dev site and organized based on file structure.
 
-Terra-dev-site will auto discover files base on either the default search patterns in the site.config or the file structure and file extensions used.
+### File Naming
 
-Add the following file to: `<package>/src/terra-dev-site/my-first-page.doc.md`
+Files need to be named using the correct 'contentExtension' and file extension to be added to the site. This will allow you to add any additional files to support your examples and only have the specified files added as entries to the site. The file name pattern is:
 
-```md
-My first page!
 ```
-
-Then re-run the site and navigate to it in your browser:
-
-```bash
-npm run start
+<filename>.<group>.<contentExtension>.<fileTypExtension>
 ```
 
-Now you should be able to see your page under the components tab.
-
-## Auto Page Discovery
-
-By default terra-dev-site will look for files in the `./src/terra-dev-site` folder in dev mode and `./lib/terra-dev-site` for prod mode. In an monorepo it will search for files in `./packages/{src,lib}/terra-dev-site`. The file directories and file names are used to build the menu navigation of the site.
+* filename - This is the name that will be added to the secondary navigation layout for the site. These are sorted alphabetically.
+* group - (optional) This allows you to sort menu items. The group is sorted alphabetically before items without a group.
+* contentExtension - This indicates which primary navigation item the content will be rendered under. The default pageTypes are `home`, `doc`, and `test`.
+* extension - .md and .mdx extensions are supported by default. Other extensions can be added by updating the projects webpack config by adding additional extensions to the resolve field.
 
 ### Directories
 
-The directories following the entry point folder (terra-dev-site by default) will be translated into part of the menu structure. The one exception is if you have a 'pageType' directory inside the entry point folder, it will be ignored. This lets you group page types together without impacting your site.
-
-### File Naming
-
-Files need to be named using the correct 'pageType' indicator and file extension to be added to the site. This will allow you to add any additional files to support your examples and only have the specified files added as entries to the site. The file name pattern is:
+The directories within the entry point folder, terra-dev-site, will be translated into the menu structure. The one exception is if you have a 'pageType' directory inside the entry point folder, it will be ignored. This lets you group page types together without impacting your site.
```

Comment:
```suggestion
The directories within the entry point folder, terra-dev-site, will be translated into the menu structure. The one exception is if you have a 'contentExtension' directory inside the entry point folder, it will be ignored. This lets you group page types together without impacting your site.
```